src/ati/terraform.py

```diff
@@ -36,21 +37,25 @@ def iterresources(filenames):
     for filename in filenames:
         with open(filename, 'r') as json_file:
             state = json.load(json_file)
+            remote_state_func = ati.remote.get_remote_func(state)
+            if remote_state_func is not None:
+                state = remote_state_func(state)
```

Comment:
minor thing, but I'm not a big fan of this mutable state; Carlton has finally gotten to me :-) Maybe something like
```
if remote_state_func:
    final_state = remote_state_func(state)
else:
    final_state = state
```
or something